spec/user_interface_spec.rb

```diff
@@ -6,21 +6,29 @@
 describe 'UserInterface' do
   let(:welcome) { "\nWelcome to Tic Tac Toe.\n\nTo play, pick a number between 1 - 9 to place a token on the board.\nThe first player with three in a row wins.\n\nIf you'd like to stop playing, you can enter 'exit' to stop.\n" }
   let (:empty_board) { ['-', '-', '-', '-', '-', '-', '-', '-', '-'] }
-  let(:moves_taken) { ['O', 'X', '-', '-', 'O', '-', '-', '-', 'X'] }
-  let(:output) { "\n  Board          Moves\n\n  |   |        1 | 2 | 3\n- + - + -      - + - + -\n  |   |        4 | 5 | 6\n- + - + -      - + - + -\n  |   |        7 | 8 | 9\n" }
-  let(:output_with_moves) { "\n  Board          Moves\n\nO | X |        1 | 2 | 3\n- + - + -      - + - + -\n  | O |        4 | 5 | 6\n- + - + -      - + - + -\n  |   | X      7 | 8 | 9\n" }
-  let(:empty_string) { '  |   |  ' }
-  let(:full_string) { 'X | X | O' }
+  let(:moves_taken) { ['O', 'X', '-', '-', 'O', '-', 'X', 'X', 'O'] }
+  let(:header) {"\e[34m     Board     \e[0m           \e[34mMoves\e[0m"}
+  let(:empty_string) { "   \e[33m | \e[0m   \e[33m | \e[0m   " }
+  let(:full_string) { "\e[35m X \e[0m\e[33m | \e[0m\e[35m X \e[0m\e[33m | \e[0m\e[32m O \e[0m" }
+  let(:normal_move_string) { " 1 \e[33m | \e[0m 2 \e[33m | \e[0m 3 " }
+  let(:dim_move_string) { "\e[2m 1 \e[22m\e[33m | \e[0m\e[2m 2 \e[22m\e[33m | \e[0m 3 " }
+  let(:row_divider) { "\e[33m--- + --- + ---\e[0m" }
+  let(:output) { "\e[34m     Board     \e[0m           \e[34mMoves\e[0m\n\n#{empty_string}          1 \e[33m | \e[0m 2 \e[33m | \e[0m 3 \n#{row_divider}      #{row_divider}"}
```

Comment:
Adding these ANSI codes into the expected messages might make the test more brittle. Maybe there is a way to keep these things separate?

Reply:
I made a [mock colorizer](https://github.com/saralein/tic-tac-toe_ruby/pull/18/files#diff-7aa5a624018bd299e043383e4af27151R4) which just returns the string.